rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -89,6 +94,13 @@ GEMFILE
 %doc %{gem_instdir}/LICENSE
 %doc %{gem_instdir}/README.md
 
+%posttrans
+%{foreman_db_migrate}
```

Comment:
not now but once the plugin will have, we'll need to add this... but I agree with the speed argument, I'll remove it